egs/librispeech/ASR/pruned_transducer_stateless7/scaling.py

```diff
@@ -994,28 +995,60 @@ def forward(ctx, x: Tensor) -> Tensor:
                 assert d_scaled.max() < 256.0
             d_int = d_scaled.to(torch.uint8)
             ctx.save_for_backward(d_int)
+        if alpha != 0:
+            y = y + alpha * x
         if x.dtype == torch.float16 or torch.is_autocast_enabled():
             y = y.to(torch.float16)
         return y
 
     @staticmethod
     def backward(ctx, y_grad: Tensor) -> Tensor:
         (d,) = ctx.saved_tensors
+        alpha = ctx.alpha
         # the same constants as used in forward pass.
         floor = -0.043637
         ceil = 1.2
         d = d * ((ceil - floor) / 255.0) + floor
-        return y_grad * d
+        ans = y_grad * (d + alpha) if alpha != 0 else y_grad * d
+        return ans, None
 
 
 class DoubleSwish(torch.nn.Module):
+    def __init__(self, alpha: float = 0.0):
```

Comment:
Shall we give it a default value `-0.05`?

Reply:
I think we should keep `alpha=0.0` as default. Since for existing repices, the models are trained with `alpha=0.0`, and their saved `state_dict` does not store this attribute.